README.md

```diff
@@ -1,30 +1,63 @@
-# Decidim::CustomProposalStates
+# Module Decidim Custom Proposal States 
 
-This module allows you to customize the proposal states.
+## Overview
 
-## Usage
+The Custom Proposal States module is a plugin for Decidim, designed to enhance the flexibility and control over the proposal answering process. This module allows administrators and valuators to create, assign, and manage custom states for proposals, providing a more tailored approach to handle community proposals.
+
+## Acknowledgments
+
+The module has been financed by the city of Lyon as well as the Civic Engagement Commission of New York, and developed by Alex Lupu for Open Source Politics. 
 
-CustomProposalStates will be available as a Component for a Participatory
-Space.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+To add this module to your Decidim instance, follow these steps:
+
+1. Add the module to your Gemfile:
+
+   ```ruby
+   gem 'decidim-custom_proposal_states'
+   ```
+
+2. Run the bundle command to install the gem:
+
+   ```bash
+   bundle install
+   ```
+
+3. Migrate your database to apply new changes:
```

Comment:
I would add as step 3 running decidim:upgrade. ( so that we copy the migrations)